app/controllers/api/auth.js

```diff
@@ -0,0 +1,46 @@
+/* eslint amd:true */
+const mongoose = require('mongoose');
+
```

Comment:
Eslint gave this message:
Expected empty line after require statement not followed by another require